src/sentry/stacktraces/processing.py

```diff
@@ -192,16 +195,15 @@ def find_stacktraces_in_data(
     rv = []
 
     def _append_stacktrace(stacktrace: Any, container: Any, is_exception: bool = False) -> None:
-        if not is_exception and (not stacktrace or not get_path(stacktrace, "frames", filter=True)):
+        frames = get_path(stacktrace, "frames", filter=True, default=())
+        if not is_exception and (not stacktrace or not frames):
             return
 
-        platforms = {
-            frame.get("platform") or data.get("platform")
-            for frame in get_path(stacktrace, "frames", filter=True, default=())
-        }
+        platforms = _get_frames_metadata(frames, data.get("platform", "unknown"))
         rv.append(
             StacktraceInfo(
                 stacktrace=stacktrace,
+                # XXX: This contains all the contents of stacktrace
```

Comment:
I'm not sure what this comment means. The container is going to be the full dictionary with `type`, `value`, `stacktrace`, `raw_stacktrace`, and `mechanism` (and possible more).